plugins/thread.py

```diff
@@ -32,13 +32,14 @@ def run(self):
         project_directory = os.path.dirname(self.process_manager.board.GetFileName())
 
         try:
+            webbrowser.open("file://%s" % (temp_dir))
```

Comment:
why is that needed? can you please remove it? I would like to open the output folder only once the process is done.

Reply:
leftover from debugging, will remove